zenith/src/main.rs

```diff
@@ -1,64 +1,93 @@
-use clap::{App, Arg, ArgMatches, SubCommand};
-use std::error;
+use std::fs;
+use std::path::{Path, PathBuf};
 use std::process::exit;
 
+use clap::{App, Arg, ArgMatches, SubCommand};
+use anyhow::Result;
+use anyhow::*;
```

Comment:
I prefer to avoid `use foo::*` most of the time. When symbols collide it can get confusing.

Reply:
I wasn't sure how to "use" macros. But it turns out to be simple:
```
use anyhow::{anyhow, bail};
```